tmpl/confirm_email.mustache

```diff
@@ -8,24 +8,29 @@
   </head>
   <body>
     <div class="container">
-      <p class="head">
-        We've sent you an email to confirm your address.
-      </p>
-      <p>
-        Use the link in that email to login to:<br>
-        <em>{{ client_id }}</em>
-      </p>
-      <p>
-        Alternatively, enter the code from the email
-        to continue in this browser tab:
-      </p>
-      <form id="form" action="/confirm" method="get">
-        <input type="hidden" name="session" value="{{ session_id }}">
-        <div class="entry">
-          <input type="text" name="code" maxlength="6" autofocus>
-          <button type="submit">Login</button>
-        </div>
-      </form>
+      <main>
+        <h1 class="head">
```

Comment:
I did not see that as issue. But testing it out it does look better aligned. https://github.com/portier/portier-broker/pull/95/commits/98cd9912e8aa1e7d8eb75388ef01560f962bb287 fixes that for me.